.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Documentation/Product Version**
+_TinyMCE 4 Documentation_ OR _TinyMCE 5 Documentation_
```

Comment:
Do we need to say this if it's specific to this repo?

Reply:
Fair. Updated